browser/DEPS

```diff
@@ -134,4 +134,8 @@ specific_include_rules = {
   "brave_content_browser_client.cc": [
     "+content/browser/renderer_host/render_frame_host_impl.h",
   ],
+  "ipfs_tab_helper_unittest.cc": [
+    "+gmock/gmock.h",
+    "+gtest/gtest.h",
+  ],
```

Comment:
Is this needed? We have other unit tests in the same folder that also include those files.

Reply:
removed it